index.js

```diff
@@ -109,8 +109,23 @@ function lookUpProfile(firstName, prop) {
 
 //Task 6
 
-const reverseString = (str) => str.split("").reverse().join("");
+const reverseString = str =>
+  str
+    .split("")
+    .reverse()
+    .join("");
 
 //Task 7
 
-const randomFraction = () => Math.round(Math.random() * 10);
\ No newline at end of file
+const randomFraction = () => Math.round(Math.random() * 10);
+
+//Task 8
+
+const factorialize = n => {
+    let result = 1;
+    while(n){
+        result *= n--
```

Comment:
Please separate this into 2 lines. It will be more readable. 
`n--;
result *= n;`